ci-operator/config/openshift/hypershift/openshift-hypershift-main__periodics.yaml

```diff
@@ -139,6 +145,63 @@ tests:
   steps:
     cluster_profile: aws
     workflow: hypershift-aws-conformance-proxy
+- as: e2e-powervs-periodic
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: hypershift
+    product: ocp
+    timeout: 1h30m0s
+    version: "4.9"
+  cron: 0 0 * * *
+  steps:
+    test:
+    - as: test-e2e
+      cli: latest
+      commands: |-
+        # Installing hypershift operator
+        bin/hypershift install --hypershift-image ${HYPERSHIFT_RELEASE_LATEST}
+
+        # Setting up credentials
+        export IBMCLOUD_CREDENTIALS=/etc/hypershift-pool-powervs-credentials/credentials
+
+        # Complie the e2e tests
+        make e2e
+
+        # Running e2e test
+        bin/test-e2e \
```

Comment:
lets also override the control plane image, ref: https://github.com/openshift/release/blob/856e3686acb14f6632959e9253e7295a03bfc203/ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.8.yaml#L99:L106

Reply:
Added cpo image as well.